src/Styleguide/Elements/Box.tsx

```diff
@@ -34,33 +35,51 @@ import {
 } from "styled-system"
 import { Flex, FlexProps } from "./Flex"
 
-const hover = css`
-  &:hover {
-    border-color: ${color("black60")};
-  }
-`
-
 export interface BorderBoxProps
   extends FlexProps,
     SpaceProps,
     MaxWidthProps,
     WidthProps,
     HeightProps {
   hover?: boolean
+  responsive?: boolean
+  hasSiblings?: boolean
 }
 
 export const BorderBox = styled(Flex).attrs<BorderBoxProps>({})`
   border: 1px solid ${color("black10")};
   border-radius: 2px;
-  ${props => props.hover && hover};
+  padding: ${space(2)}px;
   ${styledSpace};
   ${maxWidth};
   ${width};
   ${height};
+  ${({ hover }) =>
+    hover &&
+    css`
```

Comment:
Is the `css` func necessary here? I (believe) that whenever a string is interpolated within the scope of a styled components template tag function that it automatically applies `css` under the hood.

Reply:
You're right it's not strictly necessary in this instance. I would still include it though. Heres why:

e.g. if I was to change this to 

```js
  ${({ hover }) =>
    hover &&
    css`
      :hover {
        border-color: ${color("black60")};
      }
      ${media.xs`
        padding: 43px;
      `}
    `};
```

Then removing the `css` would break it because `media.xs` returns an array rather than an interpolated string. So it would be immediately rendered as `[object Object]` or something which is not correct CSS :)

I've been bitten by that a few times already, so now I use `css` whenever I interpolate non-primitive values. Think of it as a prophylactic.

Reply:
Makes sense 👍